packages/xh/build.sh

```diff
@@ -2,7 +2,7 @@ TERMUX_PKG_HOMEPAGE=https://github.com/ducaale/xh
 TERMUX_PKG_DESCRIPTION="A friendly and fast tool for sending HTTP requests"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=0.15.0
+TERMUX_PKG_VERSION=0.17.0
 TERMUX_PKG_REVISION=1
```

Comment:
Remove this revision line now that you updated the version.

Reply:
Done. I have also added manpages and completion files